### PR TITLE
Te tabbed interface

### PR DESCRIPTION
**Pull request by Testell**

### CODE REVIEW

File: app/views/users/_profile_nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<div class="row">
+  <div class="col md-6">
```
Comment (armstrca) [inline app/views/users/_profile_nav.html.erb]: `col-md-6 offset-md-3` instead of `col md-6`

**Review by armstrca — CHANGES_REQUESTED** [review]
Fix the CSS I mentioned but otherwise looks good

**Review by jessjno — APPROVED** [review]
looks good!

**Review by armstrca — APPROVED** [review]

**Review by dsimp — COMMENTED** [review]
Nice format

**Pull request merged**